openshift/oc-image-build-template.yaml

```diff
@@ -56,7 +65,43 @@ objects:
           uri: ${GIT_URI}
         type: Git
       strategy:
-        type: Docker
+        type: Source
+        sourceStrategy:
+          from:
+            kind: "DockerImage"
+            name: "quay.io/thoth-station/s2i-thoth-ubi8-py36:latest"
+          env:
+            # Enable Pipenv in OpenShift's s2i.
```

Comment:
Suggestion: Add the `THOTH_ADVISE` as a parameter, so that whenever the Thoth advice are not provided, you can toggle the parameter to 0, to have an image without an updated stack.
https://github.com/thoth-station/s2i-thoth#configuration-options

Reply:
Good idea :+1:

Reply:
Added the `THOTH_ERROR_FALLBACK` parameter as well